nginx.conf

```diff
@@ -0,0 +1,17 @@
+events {
+    # This must be here for the .conf file to be considered valid.
+}
+http {
+    server {
+        listen 80;
+        server_name techspire.csesoc.unsw.edu.au;
+
+        root /usr/share/nginx/html/;
+        index /index.html;
+
+        location / {
+            include  /etc/nginx/mime.types;
+            try_files $uri /index.html;
```

Comment:
I believe this line isn't necessary as it produces multiple HTML files at the appropriate routes (I think)

Reply:
this is true